WordPressAuthenticator/Analytics/AuthenticatorAnalyticsTracker.swift

```diff
@@ -310,14 +280,8 @@ public class AuthenticatorAnalyticsTracker {
     ///
     /// - Returns: `true` if we can track using the state machine.
     ///
-    public func canTrackInCurrentFlow() -> Bool {
-        return isInSiteAuthenticationFlowAndCanTrack()
-            || isInAppleFlowAndCanTrack()
-            || isInGoogleFlowAndCanTrack()
-            || isInMagicLinkFlowAndCanTrack()
-            || isInWPComFlowAndCanTrack()
-            || isInPrologueFlowAndCanTrack()
-            || isInKeychainFlowAndCanTrack()
+    public func canTrack() -> Bool {
+        return WordPressAuthenticator.shared.configuration.enableUnifiedAuth
```

Comment:
Since all of our flows are now enabled by this configuration, I've simplified the code.

Reply:
Closing this comment as I had to submit new code.